services/src/pro/handlers/datasets.rs

```diff
@@ -328,6 +331,7 @@ mod tests {
             exe_ctx_tiling_spec,
             TestDefault::test_default(),
             TestDefault::test_default(),
+            get_config_element::<Quota>().unwrap(),
         );
```

Comment:
does this test need to read the config.toml? can't we use a test_default here?

Reply:
the config for the tests is in Settings-test.toml. We don't do it uniformly, but e.g. the Postgres config we read from the Settings-test.toml while other config elements also implement the TestDefault trait.  But I think it's not ideal to have two places for setting the test configuration.